benchmarks/publish-subscribe/src/main.rs

```diff
@@ -113,6 +143,15 @@ struct Args {
     /// Activate full log output
     #[clap(short, long)]
     debug_mode: bool,
+    /// The cpu core that shall be used by thread 1
+    #[clap(long, default_value_t = 0)]
+    cpu_core_thread_1: usize,
+    /// The cpu core that shall be used by thread 2
+    #[clap(long, default_value_t = 1)]
+    cpu_core_thread_2: usize,
```

Comment:
How about calling this `cpu_core_participant_1` and `cpu_core_participant_2`. We could then use the same names for the multi-process benchmark.